cylc/flow/terminal.py

```diff
@@ -30,7 +30,7 @@ def is_terminal():
 def get_width(default=80):
     """Return the terminal width or `default` if it is not determinable."""
     # stty can have different install locs so don't use absolute path
-    proc = Popen(['stty', 'size'], stdout=PIPE)  # nosec
+    proc = Popen(['stty', 'size'], stdout=PIPE, stderr=PIPE)  # nosec
```

Comment:
This was printing errors when I run tests in my environment.

Reply:
Interesting, out of interest what error were you getting?

Reply:
Something about terminal type.